tests/test_processing_common.py

```diff
@@ -38,15 +39,31 @@
 from transformers.utils import is_vision_available
 
 
+global_rng = random.Random()
+
 if is_vision_available():
     from PIL import Image
 
     from transformers import CLIPImageProcessor
 
 
+# Copied from tests.models.whisper.test_feature_extraction_whisper.floats_list
+def floats_list(shape, scale=1.0, rng=None, name=None):
+    """Creates a random float32 tensor"""
+    if rng is None:
+        rng = global_rng
+
+    values = []
+    for batch_idx in range(shape[0]):
+        values.append([])
+        for _ in range(shape[1]):
+            values[-1].append(rng.random() * scale)
+
+    return values
+
+
```

Comment:
@zucchini-nlp As I noted this is copied from https://github.com/huggingface/transformers/pull/31368/files#diff-20a0d43edb95664fe346aa1952330dbc88d90b39d06a9c8bf9accfec4ac422b8

Reply:
I see, but that PR also has audio processors, while this one doesn't. We usually don't add tests before making required changes to code

Reply:
@zucchini-nlp Make sense. I just merged molbap's PR since it had some modification that need in here to pass the CI. I will remove the inappropriate commit!